md_api/src/main/java/com/wipro/ats/bdre/md/api/FetchDeploy.java

```diff
@@ -48,21 +42,22 @@ public FetchDeploy() {
             {"num", "fetch-num", "Fetch limit"}
     };
 
+    public FetchDeploy() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
+
     /**
      * @param params Sting array of Command line arguments.
      * @return
      */
     public List<ProcessDeploymentQueue> execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)